src/navigation/MainTabNavigator.tsx

```diff
@@ -165,32 +132,12 @@ const MainTabNavigator: FunctionComponent = () => {
           tabBarIcon: ExposureHistoryIcon,
         }}
       />
-      {displayReportAnIssue && (
-        <Tab.Screen
-          name={Stacks.ReportIssue}
-          component={ReportIssueStack}
-          options={{
-            tabBarLabel: t("navigation.report_issue"),
-            tabBarIcon: QuestionMarkIcon,
-          }}
-        />
-      )}
-      {displaySelfAssessment && (
-        <Tab.Screen
-          name={Stacks.SelfAssessment}
-          component={SelfAssessmentStack}
-          options={{
-            tabBarLabel: t("navigation.self_assessment"),
-            tabBarIcon: SelfAssessmentIcon,
-          }}
-        />
-      )}
       <Tab.Screen
-        name={Stacks.More}
-        component={MoreStack}
+        name={Stacks.Connect}
+        component={ConnectStack}
         options={{
-          tabBarLabel: t("navigation.more"),
-          tabBarIcon: MoreIcon,
```

Comment:
Are we ready to merge this? Wouldn't it leave the app on a non-functional state? Should we make the additions in a way that it could be there and development wise we could put it on and keep working or is this something that will wait until all elements are present to be released?

Reply:
I think we would merge it into the feature branch and then once everything is functional merge to develop.

Reply:
That might prove tricky on a long-running branch, it will take discipline to keep this branch as the main one and not using rebase on this one regularly if we have other branches spawn from this one.